copier/profiles.yml

```diff
@@ -0,0 +1,10 @@
+---
+# profile selection
+template_profile:
+  type: str
+  default: ask
```

Comment:
```suggestion
  default: recommended
```

Reply:
If we want to keep is 'ask' it should move to the top of the list.

Reply:
again, see:
- https://github.com/NLeSC/python-template/issues/526

Reply:
Cool, I must admit I have not seen this issue. They are popping like mushrooms after the rain :)